multiaccounts/database.go

```diff
@@ -76,3 +132,88 @@ func (db *Database) DeleteAccount(keyUID string) error {
 	_, err := db.db.Exec("DELETE FROM accounts WHERE keyUid = ?", keyUID)
 	return err
 }
+
+// Account images
+
+func (db *Database) GetIdentityImages(keyUID string) ([]*images.IdentityImage, error) {
+	rows, err := db.db.Query(`SELECT key_uid, name, image_payload, width, height, file_size, resize_target FROM identity_images WHERE key_uid = ?`, keyUID)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var iis []*images.IdentityImage
+	for rows.Next() {
+		ii := &images.IdentityImage{}
+		err = rows.Scan(&ii.KeyUID, &ii.Name, &ii.Payload, &ii.Width, &ii.Height, &ii.FileSize, &ii.ResizeTarget)
+		if err != nil {
+			return nil, err
+		}
+
+		iis = append(iis, ii)
+	}
+
+	return iis, nil
+}
+
+func (db *Database) GetIdentityImage(keyUID, it string) (*images.IdentityImage, error) {
+	rows, err := db.db.Query("SELECT key_uid, name, image_payload, width, height, file_size, resize_target FROM identity_images WHERE key_uid = ? AND name = ?", keyUID, it)
```

Comment:
You can use `QueryRow` here, is probably more idiomatic, so you don't have to loop, you just need to make sure you return `nil` if the error is `ErrNoRows`